userguide/tn_vcenter.rst

```diff
@@ -4,747 +4,12 @@
-   +---------------------+-------------------------------------------+
-   | Destroy Storage     | Destroy storage                           |
-   +---------------------+-------------------------------------------+
-
-
-Each role gives the user the ability to perform the functions in not
-just that role, but all of the roles that precede it in the list. For
-example, a user with a *Create Storage* role can create storage, but
-can also do *Discovery* and *Create Clones*. The vCenter administrator
-can perform all of these operations by default.
-
-
-.. note:: The vCenter plugin :guilabel:`RBAC` tab cannot create new
-   vCenter users. That must be done from vCenter
-   :guilabel:`Administration`. Hover over the
-   :guilabel:`House + lines` icon on the top section of the window to
-   view a drop-down menu of links. Click :guilabel:`Home`. From the
-   :guilabel:`Home` menu, navigate from
-   :menuselection:`Administration -->  Single Sign-On`
-   and click :guilabel:`Users and Groups`. In the :guilabel:`Users`
-   tab, click the :guilabel:`+` button to open the
-   :guilabel:`New User` window. Enter a user name and password
-   (twice) in the related fields. Optional fields can further describe
-   the user, including first and last name and an email address for
-   the account. Click :guilabel:`OK` to create the new user.
-
-
-Add a Role to an Existing vCenter User
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Click the :guilabel:`+` button to open the
-:guilabel:`Add Role Based Access Control` window. Type a user name
-in the form :samp:`{DOMAIN.NAME}\\{username}`. Click the
-:guilabel:`Assign Role` drop-down menu and choose a role for the user.
-Click :guilabel:`Add` to add the associated user and role to the list.
-If the entry does not appear immediately, click the
-:guilabel:`Refresh` button to repopulate the list.
-
-Highlight an entry in the list and click the :guilabel:`Edit` button.
-The :guilabel:`Edit` window opens, and the entry can be edited. Click
-the :guilabel:`Save` button to save any changes. Changes may take a
-moment to redraw. Click the :guilabel:`Refresh` button to repopulate
-the list.
-
-
-.. index:: vCenter Copying iSCSI Configuration
-.. _vCenter Copying iSCSI Configuration:
-
-Copying iSCSI Configuration
----------------------------
-
-The :guilabel:`TrueNAS Configuration` tab is used to duplicate some of
-the current %brand% system's iSCSI settings to another %brand% system.
-
-
-.. note:: Currently, only iSCSI targets are duplicated on the
-   destination system.
-
-
-Enter the IP address of the destination %brand% system where the
-current %brand% systems's iSCSI configuration is to be copied. Provide
-a valid user name and password from the destination system. Click
-:guilabel:`Submit` to begin duplicating the %brand% configuration to
-the destination system. This process may take some time. A popup
-window indicates when the task is complete.
+The %brand% vCenter plugin is available separately from %brand%. Please
```

Comment:
To obtain the plugin, please contact iXsystems Support at
:literal:`support@iXsystems.com` or by phone:

* 1 (855) 473-7449 option 2 (US-only toll-free)

* 1 (408) 943-4100 option 2 (local and international)